src/content/newsletters/2024-12.jsx

```diff
@@ -0,0 +1,324 @@
+				on understanding things in the tech space. Let's make some space for the
+				other parts of ourselves. In this challenge, we encourage you to spend
+				time working on things that aren't necessarily code-specific or using
+				code to improve your other hobbies and outlets.
+			</p>
+			<p>
+				Learn more about this challenge in{' '}
+				<a href="https://dev.to/virtualcoffee/monthly-challenge-creative-community-challenge-273l">
+					this blog post.
+				</a>
+			</p>
+			<p>
+				To view all of the details of this year's challenge,{' '}
+				<Link href="/monthlychallenges/dec-2024">
+					check out the December 2024 challenge page
+				</Link>
+				.
+			</p>
+			<p>And remember, we're always here to help. ❤️</p>
+
+			<h3 className="mt-4">☕🪑 Coffee Table Events</h3>
+			<ul>
+				<li>Tech Interview Study Group (Mondays at 4:00 PM ET)</li>
+				<li>
+					Accountabilibuddies (Tuesdays at 7:00 PM ET | Thursdays at 9:00 AM ET)
+				</li>
+				<li>
+					The Pack Hunt (Tandem Job Hunting) (Wednesdays at 10:00 AM ET |
+					Wednesdays at 10:00 PM ET)
+				</li>
+				<li>
+					Data Structures and Algorithms (DSA) Office Hours (Wednesdays at 4:00
+					PM ET)
+				</li>
+				<li>Feelings Friday (Fridays at 8:00 PM ET)</li>
+				<li>
+					Frontend Friday Folks fighting CSSBattle.dev (Fridays at 11:00 AM ET)
+				</li>
+			</ul>
+
+			<h3 className="mt-4">📅 Weekly Async Events</h3>
+			<ul>
+				<li>
+					<strong>Mondays:</strong> Goal-setting post in{' '}
+					<code>#goals-and-wins</code>
+				</li>
+				<li>
+					<strong>Tuesdays:</strong> Trivia in <code>#game-night</code>
+				</li>
+				<li>
+					<strong>Wednesdays:</strong> Midweek check-in in <code>#general</code>
+				</li>
+				<li>
+					<strong>Fridays:</strong>{' '}
+					<a href="https://twitter.com/VirtualCoffeeIO">X (formerly Twitter)</a>{' '}
+					chat and gratitude post in <code>#general</code>
+				</li>
+			</ul>
+			<ul>
+				<li>
+					<a href="https://github.com/orgs/Virtual-Coffee/discussions/1223">
+						Practical Deep Learning Study Group
+					</a>
+				</li>
+			</ul>
+
+			<h3 className="mt-4">☕ Official Virtual Coffee Events</h3>
+			<ul>
+				<li>
+					Virtual Coffee (Tuesdays at 9:00 AM ET | Thursdays at 12:00 PM ET)
+				</li>
+				<li>
+					Async Twitter Chat (Weekly chat begins every Friday at 9:00 AM ET)
+				</li>
+			</ul>
+
+			<p className="mt-4">
+				<strong>Note:</strong> These are the currently scheduled times for these
+				events at the time of this publication. Please check the official VC{' '}
+				<code>#announcements</code> Slack channel, or other noted channels, for
+				any updates and links to event rooms. For the full list of events, check
+				out <Link href="/events">our events page</Link>.
+			</p>
+
+			<h3 className="mt-5 mb-4">🎙️ The Podcast is Back!</h3>
+			<p>
+				Check out our latest podcasts from season 10{' '}
+				<a href="https://virtualcoffee.io/podcast">here</a>!
```

Comment:
**question:** Is there a better way to represent this link? I've been taught that linking words like "here" is a bad practice for accessibility reasons.

Reply:
Nice catch! I'll fix this!